### PR TITLE
Version names are aligned with parent.pom; dependencies and plugins a…

### PR DESCRIPTION
**Pull request by solomax**
…re updated

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -135,21 +135,21 @@
...
-		<commons-io.version>2.12.0</commons-io.version>
-		<commons-lang3.version>3.12.0</commons-lang3.version>
+		<commons-io.version>2.13.0</commons-io.version>
+		<commons-lang3.version>3.13.0</commons-lang3.version>
 		<guice.version>5.1.0</guice.version>
 		<el-impl.version>2.2.1-b05</el-impl.version>
 		<forbiddenapis.version>3.5.1</forbiddenapis.version>
 		<jacoco.version>0.8.7</jacoco.version>
-		<jackson.version>2.14.3</jackson.version>
+		<jackson.version>2.15.2</jackson.version>
```
Comment (martin-g) [inline pom.xml:152]: I have a small doubt about this one.

**Review by martin-g — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (solomax) [reply pom.xml:152]: What's wrong about it? It works as expected in wicketstuff :)

Comment (martin-g) [reply pom.xml:152]: https://issues.apache.org/jira/browse/AVRO-3754
There was a warning from a Jackson committer in the Avro project.

Comment (solomax) [reply pom.xml:152]: Sounds scary
I'll try to revert this one in several projects :(

**Pull request merged**